static/src/images/commercial/masterclasses-logo-colour.svg

```diff
@@ -0,0 +1,56 @@
+	c0.354,0,0.883,0,1.147,0.088V5.693h-0.706c-1.456,0-2.647,0.883-3.53,2.736H77.19V5.561h-0.441L71.32,6.444L71.364,6.399z
+	 M88.93,19.816c2.205,0,4.104-1.061,4.854-1.942v-1.235c-0.661,0.176-2.073,0.44-3.311,0.44c-2.205,0-3.884-1.412-3.884-4.634
+	c0-3.708,1.147-5.12,2.647-5.12c0.617,0,1.104,0.089,1.324,0.222l0.75,2.648h2.34V6.09c-0.75-0.309-2.735-0.574-4.282-0.574
+	c-4.501,0-7.412,2.647-7.412,7.149c0,4.28,2.383,7.062,6.974,7.062V19.816z M94.488,0.838v1.324l1.5,0.574v14.741l-1.323,0.707
+	v1.322h7.416v-1.322l-1.457-0.619V0h-0.353l-5.738,0.883L94.488,0.838z M106.582,7.415h1.146c1.768,0,2.295,0.882,2.295,2.559v1.766
+	l-2.646,0.441c-2.957,0.573-4.768,1.104-4.768,3.973c0,2.205,1.5,3.75,3.884,3.75c1.767,0,3.312-0.75,3.973-1.939h0.089
+	c0.223,1.455,1.457,1.896,2.869,1.896c1.102,0,2.118-0.311,2.559-0.529v-1.018l-1.457-0.705V9.666c0-3.001-2.205-4.06-5.824-4.06
+	c-2.34,0-3.883,0.617-5.162,1.059v3.531h2.206l0.883-2.781H106.582z M110.023,16.902c-0.265,0.266-0.75,0.531-1.233,0.531
+	c-0.886,0-1.767-0.443-1.767-1.986c0-1.324,0.881-1.986,1.941-2.119l1.059-0.133v3.662V16.902z M121.057,19.859
+	c4.414,0,6.18-1.941,6.18-4.768c0-2.117-0.973-3.839-3.619-4.412l-1.588-0.353c-1.502-0.31-1.897-0.795-1.897-1.589
+	c0-0.883,0.75-1.324,1.767-1.324c0.795,0,1.764,0.176,2.029,0.309l0.573,1.897h1.896V6.267c-1.103-0.441-2.735-0.706-4.322-0.706
+	c-3.973,0-5.738,1.897-5.738,4.501c0,2.029,1.146,3.973,3.619,4.59l1.677,0.353c1.456,0.312,1.767,0.706,1.767,1.767
+	c0,1.016-0.883,1.324-2.207,1.324c-0.752,0-1.854-0.178-2.207-0.441l-0.705-2.207h-1.942v3.529c1.323,0.574,3.267,0.885,4.767,0.885
+	h-0.047V19.859z M132.666,19.859c4.413,0,6.178-1.941,6.178-4.768c0-2.117-1.015-3.839-3.619-4.412l-1.588-0.353
+	c-1.5-0.31-1.898-0.795-1.898-1.589c0-0.883,0.707-1.324,1.766-1.324c0.707,0,1.678,0.176,1.986,0.309l0.574,1.897h1.897V6.267
+	c-1.017-0.441-2.647-0.706-4.239-0.706c-3.971,0-5.734,1.897-5.734,4.501c0,2.029,1.103,3.973,3.617,4.59l1.635,0.353
+	c1.457,0.312,1.764,0.706,1.764,1.767c0,1.016-0.881,1.324-2.117,1.324c-0.795,0-1.898-0.178-2.205-0.441l-0.797-2.207h-1.896v3.529
+	c1.324,0.574,3.266,0.885,4.723,0.885h-0.045V19.859z M152.259,13.152c0-5.738-1.985-7.503-6.046-7.503s-6.708,2.56-6.708,7.062
+	c0,4.635,2.384,7.191,7.062,7.191c2.472,0,4.545-1.103,5.164-1.984v-1.324c-0.885,0.311-1.896,0.44-3.664,0.44
+	c-2.517,0-3.752-1.233-3.971-3.882h8.119H152.259z M146.081,7.105c1.147,0,1.767,0.574,1.767,4.192l-3.752,0.221
+	C144.096,7.9,144.844,7.105,146.081,7.105z M157.645,19.902c4.324,0,6.09-1.984,6.09-4.854c0-2.028-0.971-3.75-3.619-4.323
+	l-1.588-0.354c-1.502-0.309-1.854-0.794-1.854-1.589c0-0.882,0.707-1.324,1.767-1.324c0.75,0,1.677,0.177,1.986,0.309l0.573,1.898
+	h1.897V6.267c-1.104-0.441-2.737-0.706-4.324-0.706c-3.973,0-5.739,1.897-5.739,4.501c0,2.029,1.149,3.973,3.621,4.59l1.675,0.353
+	c1.457,0.312,1.768,0.706,1.768,1.767c0,1.016-0.883,1.324-2.207,1.324c-0.75,0-1.766-0.178-2.207-0.441l-0.707-2.207h-1.941v3.529
+	c1.324,0.574,3.311,0.885,4.768,0.885L157.645,19.902z"/>
+</svg>

```

Comment:
I use an app called SVGCleaner to do my decimal point optimisation. It will also remove most of the Illustrator related junk http://sourceforge.net/projects/svgcleaner/